test/roundtrip/try.txt

```diff
@@ -0,0 +1,30 @@
+;;; TOOL: run-roundtrip
```

Comment:
Will add tests once wasm-objdump is made aware of exceptions (i.e. in a separate PR).